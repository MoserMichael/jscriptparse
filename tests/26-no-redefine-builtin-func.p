a="42"
println("value of variable a: {a}")

def a() { return 43 }
println("value of function call: {a()}")

# this is an error - can't redefine built-in functions
def len() { return 42 }
