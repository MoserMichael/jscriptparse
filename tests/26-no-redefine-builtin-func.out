value of variable a: 42
value of function call: 43
Error: Can't redefine built-in function len
